### PR TITLE
Migrate Google ads example to new design AIP-47

### PR DESCRIPTION
**Pull request by chenglongyan**
related: #22447, #22430

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/system/providers/google/ads/example_ads.py [hunk 1]
```diff
@@ -61,28 +68,58 @@
...
     "metrics.cost_micros.value",
 ]
-
 # [END howto_google_ads_env_variables]
 
 with models.DAG(
-    "example_google_ads",
-    schedule_interval=None,  # Override to match your needs
+    DAG_ID,
+    schedule_interval='@once',
```
Comment (turbaszek) [inline tests/system/providers/google/ads/example_ads.py:75]: Is this change necessary?

Comment (chenglongyan) [reply tests/system/providers/google/ads/example_ads.py:75]: This is mentioned in the AIP-47 **Design details**.

**Review by potiuk — APPROVED** [review]
